dictionary.json

```diff
@@ -113,14 +108,14 @@
     "affected_code": {
       "caption": "Affected Code",
       "description": "List of Affected Code objects that describe details about code blocks identified as vulnerable.",
-      "is_array": true,
-      "type": "affected_code"
+      "type": "affected_code",
```

Comment:
I wish all keys were alphabetically sorted. The change have alphabetical sorting for event names but not attributes.